### ISSUE TITLE
app-emacs/idris-mode fails to install, Cannot open load file: prop-menu

### ISSUE THREAD

**Issue by NobbZ** [gentoo-haskell/gentoo-haskell#609]
I tried to `emerge app-emacs/idris-mode` and it failed with the following messages (stripped irrelevant parts):

```text
In toplevel form:
idris-commands.el:31:1:Error: Cannot open load file: No such file or directory, prop-menu

In toplevel form:
idris-hole-list.el:27:1:Error: Cannot open load file: No such file or directory, prop-menu

In toplevel form:
idris-info.el:28:1:Error: Cannot open load file: No such file or directory, prop-menu

In toplevel form:
idris-mode.el:20:1:Error: Cannot open load file: No such file or directory, prop-menu

In toplevel form:
idris-repl.el:29:1:Error: Cannot open load file: No such file or directory, prop-menu

In toplevel form:
idris-tests.el:27:1:Error: Cannot open load file: No such file or directory, prop-menu

In toplevel form:
idris-tree-info.el:29:1:Error: Cannot open load file: No such file or directory, prop-menu
```

I do think it is because of a not specified (and not available in portage) dependency [`prop-menu`](https://github.com/david-christiansen/prop-menu-el) (emacs package). The dependency is missed only in a nota-bene in the source repositories README:

> Alternatively, download the elisp files, and place them somewhere in your load path. If you're installing manually, then you also need to install the `prop-menu` package, which idris-mode uses to create contextual menus. Installation from MELPA will automatically install dependencies.

Even if I will do a MELPA install (easier to configure and maintain over various install locations sharing the same dot-files) I think it should be fixed if possible at all.

Perhaps it would even make sense to version this package as it is versioned on MELPA/MELPA-stable?

**Comment by trofi** [comment]
Yeah, having non-git `idris-mode` ebuild and it's deps satisfied would be nice.

**Comment by trofi** [comment]
I've pushed `idris-mode-0.9.18` which does not seem to require prop-mode and at least does source highlighting.

**Comment by ezzieyguywuf** [comment]
@trofi I realize this is a few years old now, but I figured it'd be worth asking: is this bug still relevant?

1. I was able to succesfully emerge `app-emacs/idris-mode-0.9.18`
2. It also seems that theres a (1.0 release)[https://github.com/idris-hackers/idris-mode/releases/tag/1.0], though I didn't need that in order to  merge the package.

I can't speak to how well the idris mode does/doesn't work since I'm not that familiar with emacs, but the original bug report seems to state that the package was not mergeable at all.
